packages/connect/src/base/entry.ts

```diff
@@ -300,7 +317,7 @@ export class EntryNodes extends EventEmitter {
     }
     log(`Updating list of used relay nodes ...`)
     const nodesToCheck = this.filterUncheckedNodes()
-    const TIMEOUT = 3e3
+    const TIMEOUT = 5e3
```

Comment:
Please document timeout and reasoning for it.

Reply:
changed